hapi/build.gradle.kts

```diff
@@ -29,27 +29,19 @@ tasks.withType<JavaCompile>().configureEach {
     options.compilerArgs.add("-Xlint:-exports,-deprecation,-removal")
 }
 
-// Add downloaded HAPI repo protobuf files into build directory and add to sources to build them
-tasks.cloneHederaProtobufs {
-    // uncomment below to use a specific tag
-    // tag = "v0.53.0"
-    // uncomment below to use a specific branch
-    branch = "main"
-}
-
 sourceSets {
     main {
         pbj {
-            srcDir(tasks.cloneHederaProtobufs.flatMap { it.localCloneDirectory.dir("services") })
-            srcDir(tasks.cloneHederaProtobufs.flatMap { it.localCloneDirectory.dir("streams") })
-            srcDir(tasks.cloneHederaProtobufs.flatMap { it.localCloneDirectory.dir("platform") })
-            srcDir(tasks.cloneHederaProtobufs.flatMap { it.localCloneDirectory.dir("block") })
+            srcDir(layout.projectDirectory.dir("hedera-protobufs/services"))
+            srcDir(layout.projectDirectory.dir("hedera-protobufs/streams"))
+            srcDir(layout.projectDirectory.dir("hedera-protobufs/block"))
+            srcDir(layout.projectDirectory.dir("hedera-protobufs/platform"))
         }
         proto {
-            srcDir(tasks.cloneHederaProtobufs.flatMap { it.localCloneDirectory.dir("services") })
-            srcDir(tasks.cloneHederaProtobufs.flatMap { it.localCloneDirectory.dir("streams") })
-            srcDir(tasks.cloneHederaProtobufs.flatMap { it.localCloneDirectory.dir("platform") })
-            srcDir(tasks.cloneHederaProtobufs.flatMap { it.localCloneDirectory.dir("block") })
+            srcDir(layout.projectDirectory.dir("hedera-protobufs/services"))
+            srcDir(layout.projectDirectory.dir("hedera-protobufs/streams"))
+            srcDir(layout.projectDirectory.dir("hedera-protobufs/block"))
```

Comment:
This line might need to wait a bit.
The block stream HIP isn't even published yet, and block streams protobufs may change substantially.

Reply:
Makes sense...for now, since Services is aiming to ship preview block streams to previewnet in `0.54`, for convenience can we keep these here until their final destination is known?

Reply:
We likely will need to, as I don't know when we'll have a solution for packaging protobufs from other repos.
We may have a few mainnet releases between now and when we all work out a functional solution for the need to split ownership of protobufs among multiple code repos.